README.md

```diff
@@ -78,6 +100,9 @@ Under data, we can load the config file under its desired name, and we can attac
 
 This will put /etc/nginx/nginx.conf into the docker container with that absolute path, equivalent to copying over a local nginx.conf file in a Dockerfile. 
 
+## InitContainers
+InitContainers are generally lightweight containers that contain a few simple instructuions. The initContainers must run and successfully exit before the pod can start. If it fails, Kubernetes will keep trying to restart the initContainers until it is successful. You can have multiple initContainers, and they will simply execute one by one in the order they are defined. We will look at specifi examples of how we use initContainers with [nginx](#nginx-initcontainer) and [redis](#redis-initcontainer) in the secions below.
```

Comment:
specifi -> specific typo
maybe change "before the pod can start" to "before a deployment's pods can start"? Might be a little more clear